samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1920,6 +1920,54 @@
         }
       }
     },
+    "SESBulkTemplatedCrudPolicy_v2": {
```

Comment:
I believe we decided that `_v2` should be a new file, not a new policy

Reply:
going to double check this

Reply:
hmm actually I thought it was just a new policy with _v2 in the same file rather than a new file. (I was thinking each was in its own file rather than all in one big file, not sure we really want to copy paste the whole thing to add one more policy).  Others should weigh in.

Reply:
This is a regular additive change, I see no benefit to a separate file.

Reply:
There is a difference if we set up an automated check for policy templates as we have for profiles. Having all versions in one file will makes running those checks much harder in case an older version is considered problematic, and a new one has a fix and both are in the same file.

Reply:
I think that's solvable - same file is going to be a lot simpler, although as we scale some sort of breaking apart (perhaps by logical groupings like service) will be easier to comprehend and to maybe add the possibility of lazy loading if/when performance becomes an issue.